samples/drivers/fpga/fpga_controller/src/main.c

```diff
@@ -23,6 +24,14 @@ void main(void)
 			       PAD_SMT_DISABLE | PAD_REN_DISABLE | PAD_SR_SLOW |
 			       PAD_CTRL_SEL_AO_REG); /* Enable green led */
 
+#if CONFIG_SHELL
+	printk("Address of the bitstream (red): %p\n", &axFPGABitStream_red);
+	printk("Address of the bitstream (green): %p\n", &axFPGABitStream_green);
+	printk("Size of the bitstream (red): %d\n", sizeof(axFPGABitStream_red));
+	printk("Size of the bitstream (green): %d\n", sizeof(axFPGABitStream_green));
+
+
```

Comment:
I don't think we need these whitespaces here

Reply:
Yes, Removed them.